python/sqlflow_submitter/xgboost/train.py

```diff
@@ -166,21 +166,72 @@ def train(datasource,
         print("Evaluation result: %s" % re)
 
     if rank == 0:
-        # TODO(sneaxiy): save pmml as well
-        model_name = "my_model"
-        bst.save_model(model_name)
+        filename = "my_model"
+        save_local_file(bst, model_params, filename)
 
         if is_pai and len(oss_model_dir) > 0:
-            save_model(oss_model_dir, model_name, model_params, train_params,
+            save_model(oss_model_dir, filename, model_params, train_params,
                        feature_metas, feature_column_names, label_meta,
                        feature_column_code)
 
 
-def save_model(model_dir, model_name, model_params, train_params,
-               feature_metas, feature_column_names, label_meta,
-               feature_column_code):
-    model.save_file(model_dir, model_name)
-    # TODO(sneaxiy): save pmml as well
+def save_local_file(booster, model_params, filename):
```

Comment:
`save_model_local`?

Reply:
Done. Changed to `save_model_to_local_file` .